### ISSUE TITLE
[S2S] Unclear duplicate detection message when processing a shared contact whose source case belongs to the same person

### ISSUE THREAD

**Issue by adinaflorea9** [hzi-braunschweig/SORMAS-Project#11292]
### Problem Description

When trying to accept a contact shared via S2S in the receiving system, there is a duplicate detection pop-up that gets triggered that is not completely clear, when the contact shared has a source case that belongs to the same person.

Steps to reproduce the described behavior:
1. Create a case and a contact for the same person;
2. Set the case as the source case of the contact;
3. Share via S2S the case and then the contact;
4. In the receiving system, accept the case first (it is mandatory to accept the case first);
5. Then click to accept the contact.

Actual behavior:
A duplicate detection pop-up is triggered that in one part mentions that a similar case was found (even though there is only 1 case in the system for that person) and then in the second part it mentions that the user might have to convert the contact.
![image](https://user-images.githubusercontent.com/96909231/211003920-ee240a44-f992-4446-a4e7-b599eee37d28.png)

It is unclear why the message first mentions that there is at least one similar case in the system because at step 5 the user is trying to accept the contact and not the case.

@SahaLinaPrueger  Would you please be able to perhaps reproduce the behavior and let me know what you think about this duplicate detection message and its contents? (the one that appears immediately after step 5)

I also added a GIF of the steps that I took and the behavior:
![S2S - error](https://user-images.githubusercontent.com/96909231/211004298-9e247a2c-47fa-47d5-bbbf-11dd330376d5.gif)

### Additional Information

Logged the behavior as a task and added it to the investigation board as this is not necessarily a bug, or a finding since investigation needs to be done to determine if the behavior is the expected one, and if not, is this a bug or a change. Furthermore, the described issue does not impact the actual functionality of processing share requests as they can still be accepted or rejected.

**Comment by SahaLinaPrueger** [comment]
The duplicate detection is specified here: https://github.com/hzi-braunschweig/SORMAS-Project/issues/9527#issuecomment-1265442536 _accepting contact_ Point 3 is about the message you found and was specified like this.
To be honest I would really like to re-define the duplicate detection because of other points. But that would be a change.

**Comment by adinaflorea9** [comment]
@SahaLinaPrueger Thank you for looking over this! 
So from what I understand from your comment to this ticket, and from the comment from ticket #9527 that message would be supposed to appear when a similar case already exists in the receiving system.

But what is confusing for me right now and why I opened this investigation task is the fact that in the receiving system, for the person associated, there is currently only 1 case (the one that was sent at step 3, and that case is the source case of the contact, and not its resulting case). So when I accept that contact, there are no 2 cases that might be similar and I don't quite get why that message would then mention that there is a similar case. Since this contact that was shared only had a source case and no resulting case.

**Comment by SahaLinaPrueger** [comment]
@adinaflorea9 Ok, now I know what you mean. Thanks for the explanation. Then the message itself is not well worded. The message is supposed to be about the fact that not only is a similar person with a case as an entity already in the system, but that this case additionally meets the time specification, so it is clear to the user that this accepted contact should most likely be converted to a case and the two then existing cases would need to be merged. So a more understandable wording would be appropriate.

**Comment by adinaflorea9** [comment]
@SahaLinaPrueger  Thank you for taking the time to look over this again and for explaining this to me! In that case, I'd like for the message to be updated to have better wording if this is ok for you as well.

Is there any form for the message that you would prefer? If so, you could provide an example and the ticket can be updated and converted so that the message can be updated.

**Comment by SahaLinaPrueger** [comment]
> Is there any form for the message that you would prefer?

_There is at least one similar person in your system. This person has a case as an entity with the same disease and a report date close in time.
This is why, after accepting the request, it may be necessary to perform a manual conversion of the accepted contact and to merge duplicate persons and cases_

Additional information: To be honest i would like to delete the recommendation "This is why, after accepting the request, it may be necessary to perform...", because we don't want to be a substitute for the independent thinking of the user and also we are not responsible for the further actions. But that would then have to be done uniformly for all messages. I will talk to my team because of that and give you an answer the next week. 

Additional information 2: duplicate detection for S2S was implemented before person merge was implemented. So the given recommendations nowadays are incomplete or false and have to be updated.

Additional information 3: duplicate detection for cases does not fit good for S2S because cases need the same place of stay region or the same responsible region to be detected as a duplicate.

**Comment by SORMAS-SahaLinaPrueger** [comment]
@adinaflorea9 I talked with the team about the additional information 1 and 2 and we found a solution. If you want I can write down our results/specification in a new change issue?

**Comment by adinaflorea9** [comment]
@SORMAS-SahaLinaPrueger Thank you, yes, that would be great!

**Comment by adinaflorea9** [comment]
Moved the ticket to waiting until the new issue is specified by @SORMAS-SahaLinaPrueger
